devbook.xsl

```diff
@@ -542,16 +554,16 @@
                 </button>
               </div>
               <div class="collapse navbar-collapse" id="gw-toolbar">
```

Comment:
Here it looks like this: https://dev.gentoo.org/~ulm/screenshots/navigation.png
So especially, the dropdown list gets expanded, and the little icons before "Home", "Virtuals" and "Ebuild File Format" are missing (because they're part of Bootstrap).

Reply:
The list of parents also wasn't optimal in offline mode, because it was expanded into a list.

I've now replaced the whole header by a very simple navigation, only with "previous", "up", and "next" links. It was like that in the old version:
https://web.archive.org/web/20141026121545/http://devmanual.gentoo.org/